e2e/include/idata.h

```diff
@@ -26,7 +26,7 @@ typedef struct
     // the buffer will be freed by the idata free function
     const char *path;
     // the buffer will be freed by the idata free function
-    astarte_individual_t individual;
+    astarte_data_t individual;
```

Comment:
nit: change also the field to data
```suggestion
    astarte_data_t data;
```